### PR TITLE
Get the FDL app running with the new provisioning profile, entitlemen…

### PR DESCRIPTION
**Pull request by dmandar**
…ts and info.plist entries.

### CODE REVIEW

File: FirebaseDynamicLinks/Tests/Sample/FDLBuilderTestAppObjC/SceneDelegate.h [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2018 Google
```
Comment (paulb777) [inline FirebaseDynamicLinks/Tests/Sample/FDLBuilderTestAppObjC/SceneDelegate.h]: 2020

File: FirebaseDynamicLinks/Tests/Sample/FDLBuilderTestAppObjC/SceneDelegate.m [hunk 2]
```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2018 Google
```
Comment (paulb777) [inline FirebaseDynamicLinks/Tests/Sample/FDLBuilderTestAppObjC/SceneDelegate.m]: 2020

File: FirebaseDynamicLinks/Tests/Sample/FDLBuilderTestAppObjC/SceneDelegate.h (hunk 1 above)
Comment (dmandar) [reply FirebaseDynamicLinks/Tests/Sample/FDLBuilderTestAppObjC/SceneDelegate.h]: Done.

File: FirebaseDynamicLinks/Tests/Sample/FDLBuilderTestAppObjC/SceneDelegate.m (hunk 2 above)
Comment (dmandar) [reply FirebaseDynamicLinks/Tests/Sample/FDLBuilderTestAppObjC/SceneDelegate.m]: Done.

**Pull request merged**